physical/raft/raft.go

```diff
@@ -594,8 +542,13 @@ func (b *RaftBackend) Close() error {
 		return err
 	}
 
-	if err := b.stableStore.(*raftboltdb.BoltStore).Close(); err != nil {
-		return err
+	// This relies on logStore == stableStore and not having any middleware
+	// wrappers. If these assumptions change, it's possible this will break,
+	// and we should adjust accordingly at that time.
```

Comment:
```suggestion
	// This relies on logStore == stableStore and not having any middleware
	// wrappers around the stableStore (the logStore is always wrapped
	// which is why we call close on it rather than stableStore so all 
	// middleware see the Close too). If these assumptions change,
	// it's possible this will break, and we should adjust accordingly.
```